src/Runtime/Distributions/Automata/Automaton.cs

```diff
@@ -786,6 +786,11 @@ public string ToString(Action<TElementDistribution, StringBuilder> appendElement
             }
 
             var builder = new StringBuilder();
+            if (this.LogValueOverride.HasValue)
+            {
+                builder.Append(this.LogValueOverride);
```

Comment:
I think, we should add some separator after the override. Like `builder.Append(':');`